Makefile.d/k8s.mk

```diff
@@ -98,6 +98,33 @@ k8s/vald/delete:
 	kubectl delete -f $(TEMP_DIR)/vald/templates/agent
 	rm -rf $(TEMP_DIR)
 
+.PHONY: k8s/multi/vald/deploy
+## deploy multiple vald sample clusters to k8s
+k8s/multi/vald/deploy:
+	-@kubectl create ns vald-01
+	-@kubectl create ns vald-02
+	-@kubectl create ns vald-03
+	helm install vald-cluster-01 charts/vald \
+		-f ./charts/vald/values/multi-vald/dev-vald-with-mirror.yaml \
+		-f ./charts/vald/values/multi-vald/dev-vald-01.yaml \
+	    -n vald-01
+	helm install vald-cluster-02 charts/vald \
+		-f ./charts/vald/values/multi-vald/dev-vald-with-mirror.yaml \
+		-f ./charts/vald/values/multi-vald/dev-vald-02.yaml \
+	    -n vald-02
+	helm install vald-cluster-03 charts/vald \
+		-f ./charts/vald/values/multi-vald/dev-vald-with-mirror.yaml \
+		-f ./charts/vald/values/multi-vald/dev-vald-03.yaml \
+		-n vald-03
```

Comment:
It would be better to apply this command for multi-Kubernetes clusters, too, because this implementation can be used for multi-Vald clusters whose Kubernetes cluster is not the same as each other.

So, I think you should define namespace as a Makefile variable.
And more, can you add the kubeconfig variable as the same above?

Reply:
Wait for your response :)

Reply:
Thank you for your comment 🙏  Yes, I agree with your opinion!
However, when deploying to a different cluster, it is necessary to change the cluster ingress configuration and mirror client configuration ...etc, and it will be complicated and difficult to do so using only the makefile command... Therefore, it would be better to deploy this command to multiple namespaces in a local single cluster, and to describe in the documentation for multiple clusters and explain it to users 🤔 

> I think you should define namespace as a Makefile variable.

OK, I will fix it later.

Reply:
I fixed 🙏 
https://github.com/vdaas/vald/pull/1949/commits/96edc39f3b7809b1ea8cbc607abfe3eea6a5f158